serverless/event_sources/knative-event-sources.adoc

```diff
@@ -1,25 +1,23 @@
 include::modules/serverless-document-attributes.adoc[]
 [id="knative-event-sources"]
-= Getting started with event sources
+= Understanding event sources
 include::modules/common-attributes.adoc[]
 :context: knative-event-sources
 
 toc::[]
 
-An _event source_ is an object that links an event producer with an event _sink_, or consumer. A sink can be a Knative service, channel, or broker that receives events from an event source.
+A Knative _event source_ can be any Kubernetes object that generates or imports CloudEvents, and relays those events to another endpoint, known as a xref:../../serverless/knative_eventing/serverless-event-sinks.adoc#serverless-event-sinks[_sink_]. Sourcing events is critical to developing a distributed system that reacts to events.
```

Comment:
Could CloudEvents become something like "`CloudEvent` objects" or something like that? IIRC, our in-house guidelines discourage using terms like that as nouns.

Reply:
There were a few other places this was used in the docs, so I've updated to either "cloud events" or `CloudEvent` object as appropriate. FWIW I really don't like this guideline because it makes our docs a lot clunkier to read but I guess while we're part of OCP docs we're tied to it.